### PR TITLE
Minor updates.

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: include/hpp/core/steering-method/straight.hh [hunk 1]
```diff
@@ -21,12 +21,8 @@
 
 # include <hpp/core/config.hh>
 # include <hpp/core/fwd.hh>
-# include <hpp/core/problem.hh>
 # include <hpp/core/steering-method/fwd.hh>
 # include <hpp/core/steering-method.hh>
-# include <hpp/core/straight-path.hh>
-# include <hpp/core/distance.hh>
-# include <hpp/core/config-projector.hh>
```
Comment (jmirabel) [inline include/hpp/core/steering-method/straight.hh]: This reduction of header dependency causes a compilation problem in hpp-rbprm. I will provide a fix there.

**Comment by nim65s** [comment]
@jmirabel : can we merge this for the release ?

**Comment by nim65s** [comment]
I see, this was waiting for https://github.com/humanoid-path-planner/hpp-rbprm/pull/59

**Comment by jmirabel** [comment]
The release will include the new version of hpp-fcl ?
If yes, then, we can merge this and the related PRs. Everything works fine on my laptop.

**Comment by nim65s** [comment]
It will :)

Thanks !

**Pull request merged**